### DISCUSSION TITLE
Installing VSCode Snippets from Market

### DISCUSSION THREAD

**Discussion by bennypowers** [L3MON4D3/LuaSnip#472] [Q&A] [answered]
Hello!

So, let's say I:

```lua
require 'luasnip.loaders.from_vscode'.lazy_load { paths = {
  -- Path to the built project, perhaps in your `node_modules`
  "~/Developer/my-vscode-snippets-package/"
} }
```

Ok, fine, works locally, *but* requires me to manually download the snippets package, build it, and keep track of which dir it's in.

Is it possible to rather install vscode snippets by marketplace url, and have luasnip download and install it in `~/.local`?

**Comment by leiserfg** [comment, marked as answer]
Well, doing that is potentially illegal (https://github.com/microsoft/vscode/issues/31168) besides that it will be hard to do without adding several dependencies because nvim and lua don't have built-in HTTP client nor a `zip` file extractor.

**Reply by bennypowers** [reply]
I weep for my generation

Thanks for the answer!
